packages/react-core/src/components/ToggleGroup/ToggleGroupItem.tsx

```diff
@@ -53,8 +56,13 @@ export const ToggleGroupItem: React.FunctionComponent<ToggleGroupItemProps> = ({
         disabled={isDisabled}
         id={buttonId}
       >
-        {icon && <ToggleGroupItemElement variant={ToggleGroupItemVariant.icon}>{icon}</ToggleGroupItemElement>}
+        {icon && iconPosition === 'start' && (
+          <ToggleGroupItemElement variant={ToggleGroupItemVariant.icon}>{icon}</ToggleGroupItemElement>
```

Comment:
nit: we could extract this ToggleGroupItemElement to a variable to not have it duplicated